upgrade_guide/migrating_from_astyanax/language_level_changes/README.md

```diff
@@ -0,0 +1,66 @@
+             |                                                                                            |
+One Thrift   |                                                                                           -/
+COLUMNFAMILY |          
+             |  
+             |  /------------\              /---------------+---------------+---------+
+             |  |            |              |     Col 1     |     Col 4     |         |
+             |  | Key : 2    | ---------->  +---------------+---------------+ ...     |
+             |  |            |              |     Val : 'a' |     Val : 'b' |         |
+             |  \------------/              \---------------+---------------+---------+
+             |  
+             \- 
+
+
+                      -----------------------------------------------------------------------
+
+
+                                                     CQL
+                                                     
+             /-                              
+             |                               
+             |  /--------------------+---------------------------+-----------------------+----------------------+-----------------------------\
+             |  |       key          |           Col1            |         Col2          |         Col3         |            Col4             |  
+             |  +--------------------+---------------------------+-----------------------+----------------------+-----------------------------+ -\
+             |  | cRED   1           |   c1AB     'a'            | cFA0     'b'          |          'c'         |             null            |  +--> One CQL
+   One CQL   |  +--------------------+---------------------------+-----------------------+----------------------+-----------------------------+ -/    ROW
+   TABLE     |  |        2           |             'a'           |          null         |          null        |             'b'             |
+             |  +--------------------+---------------------------+-----------------------+----------------------+-----------------------------+
+             |  |       ...          |            ...            |          ...          |          ...         |             ...             |
+             |  +--------------------+---------------------------+-----------------------+----------------------+-----------------------------+
+             \-
+```
+
+Some of the columns of a *CQL Table* have a special role that is specifically
+related to the *Cassandra* architecture. Indeed, the *Row key* of the *Thrift Row*,
+becomes the *Partition Key* in the *CQL Table*, and can be composed of 1 or multiple
+*CQL columns* (the key column in Figure 1). The *“Column”* part of the Column-value
+component in a *Thrift Row*, becomes the *Clustering ColumnKey* in *CQL*, and can
+also be composed of multiple columns (in the figure, column1 is the only column 
+composing the *Clustering ColumnKey*).
```

Comment:
Hm I see what you mean, was it possible to have two "Col:1" for the same Row Key in Thrift? I don't see how I can do that, with Astyanax at least, I tried inserting twice for the Same Row Key and The Same "Col:x" 2 different values, and one erases the other, since the "Col:" is part of the Clustering key